File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/LabTestsBatchItemCategoriesItem.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class LabTestsBatchItemCategoriesItem(
    @JsonProperty("ProductCategory")
    val productCategory: LabTestsBatchItemCategoriesItemProductCategory? = null,
    @JsonProperty("ProductCategoryId")
    val productCategoryId: String? = null,
    @JsonProperty("RequireLabTestBatch")
    val requireLabTestBatch: Boolean? = null
)

File: sdks/thunkmetrc-go/wrapper/models/lab_tests_batch_item_categories_item.go
package models

type LabTestsBatchItemCategoriesItem struct {
    ProductCategory LabTestsBatchItemCategoriesItemProductCategory `json:"ProductCategory,omitempty"`
    ProductCategoryId string `json:"ProductCategoryId,omitempty"`
    RequireLabTestBatch bool `json:"RequireLabTestBatch,omitempty"`
}
